src/main/java/com/uber/jenkins/phabricator/tasks/NonDifferentialHarbormasterTask.java

```diff
@@ -87,7 +88,7 @@ protected void execute() {
         try {
             harbormaster.sendHarbormasterUri(phid, buildUrl);
             // Only send pass/fail, since coverage and unit aren't viewable outside of differentials
-            harbormaster.sendHarbormasterMessage(phid, pass, null, null, null);
+            harbormaster.sendHarbormasterMessage(phid, pass ? MessageType.pass : MessageType.fail, null, null, null);
```

Comment:
Should non-differential builds also respect sending "work" updates?

Reply:
That is a good question. I think the answer is yes as the Harbormaster status would be incorrect for those builds as well. Will fix.

Reply:
Fixed